apps/app/src/app/pages/(components)/components/(radio-group)/radio-group--card.example.ts

```diff
@@ -2,113 +2,108 @@ import { Component } from '@angular/core';
+					/>
+				</svg>
+				PayPal
+			</hlm-radio>
+			<hlm-radio value="apple" [class]="cardClass">
+				<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" class="mb-3 size-6">
+					<title>Apple</title>
+					<path
+						d="M12.152 6.896c-.948 0-2.415-1.078-3.96-1.04-2.04.027-3.91 1.183-4.961 3.014-2.117 3.675-.546 9.103 1.519 12.09 1.013 1.454 2.208 3.09 3.792 3.039 1.52-.065 2.09-.987 3.935-.987 1.831 0 2.35.987 3.96.948 1.637-.026 2.676-1.48 3.676-2.948 1.156-1.688 1.636-3.325 1.662-3.415-.039-.013-3.182-1.221-3.22-4.857-.026-3.04 2.48-4.494 2.597-4.559-1.429-2.09-3.623-2.324-4.39-2.376-2-.156-3.675 1.09-4.61 1.09zM15.53 3.83c.843-1.012 1.4-2.427 1.245-3.83-1.207.052-2.662.805-3.532 1.818-.78.896-1.454 2.338-1.273 3.714 1.338.104 2.715-.688 3.559-1.701"
+					/>
+				</svg>
+				Apple
+			</hlm-radio>
 		</hlm-radio-group>
 	`,
 })
 export class RadioGroupCardComponent {
 	public payment = 'card';
+
+	public readonly cardClass = hlm(
+		'block space-x-0',
+		// base card styles for the label
+		'[&>[data-slot=label]]:flex [&>[data-slot=label]]:flex-col [&>[data-slot=label]]:items-center [&>[data-slot=label]]:justify-between [&>[data-slot=label]]:bg-popover [&>[data-slot=label]]:rounded-md [&>[data-slot=label]]:border-2 [&>[data-slot=label]]:border-muted [&>[data-slot=label]]:p-4',
+		// hover styles
+		'[&>[data-slot=label]]:hover:bg-accent [&>[data-slot=label]]:hover:text-accent-foreground',
+		// highlight checked radio
+		'[&>[data-slot=label]]:data-[checked=true]:border-primary',
+	);
 }
 
 export const cardCode = `
 import { Component } from '@angular/core';
 import { FormsModule } from '@angular/forms';
-import { provideIcons } from '@ng-icons/core';
+import { NgIcon, provideIcons } from '@ng-icons/core';
 import { lucideApple, lucideCreditCard } from '@ng-icons/lucide';
-import { BrnRadioComponent } from '@spartan-ng/brain/radio-group';
+import { hlm } from '@spartan-ng/brain/core';
 import { HlmIconDirective } from '@spartan-ng/ui-icon-helm';
-import { HlmRadioDirective, HlmRadioGroupComponent } from '@spartan-ng/ui-radiogroup-helm';
+import { HlmRadioComponent, HlmRadioGroupComponent } from '@spartan-ng/ui-radiogroup-helm';
 
 @Component({
 	selector: 'spartan-radio-card-preview',
 	standalone: true,
 	providers: [provideIcons({ lucideCreditCard, lucideApple })],
-	imports: [FormsModule, BrnRadioComponent, HlmRadioDirective, HlmRadioGroupComponent, HlmIconDirective],
+	imports: [FormsModule, HlmRadioComponent, HlmRadioGroupComponent, NgIcon, HlmIconDirective],
```

Comment:
I don't think we are using lucideApple icon anymore, so we can probably remove it. Alternatively we could use the Apple and Paypal icons from @ng-icons/remixicon rather than inlining the SVGs, whatever you prefer.

Reply:
Good idea! I updated the icons and import them from @ng-icons/remixicon, better than inline SVGs for the example